packages/@aws-cdk/aws-lambda-nodejs/lib/types.ts

```diff
@@ -66,6 +66,65 @@ export interface BundlingOptions {
    */
   readonly keepNames?: boolean;
 
+  /**
+   * Normally the esbuild automatically discovers `tsconfig.json` files and reads their contents during a build.
+   *
+   * However, you can also configure a custom `tsconfig.json` file to use instead.
+   *
+   * This can be useful if you need to do multiple builds of the same code with different settings:
+   *
+   * @default - tsconfig.json
+   */
+  readonly tsConfig? : string
+
+  /**
+   * This option tells esbuild to write out a JSON file with metadata about the build.
+   * The metadata in this JSON file follows this schema (specified using TypeScript syntax):
+   * ```typescript
+   *  {
+   *     outputs: {
+   *          [path: string]: {
+   *            bytes: number
+   *            inputs: {
+   *              [path: string]: { bytesInOutput: number }
+   *            }
+   *            imports: { path: string }[]
+   *            exports: string[]
+   *          }
+   *        }
+   *     }
+   * }
+   * ```
+   * This data can then be analyzed by other tools. For example,
+   * bundle buddy can consume esbuild's metadata format and generates a treemap visualization
+   * of the modules in your bundle and how much space each one takes up.
+   * @see https://esbuild.github.io/api/#metafile
+   * @default - false
+   */
+  readonly metaFile?: boolean
```

Comment:
Out of curiosity, what is your use case here? and does it make sense to ship this metafile in the code bundle (zip)?

Reply:
Agreed, I don't want to send the file in bundle (zip). maybe we need to exclude the file from the bundled zip.

Reply:
Although it just meant to just view the stats of bundled during development